rls.toml

```diff
@@ -0,0 +1 @@
+clippy_preference = "on"
```

Comment:
Is this file actually currently being read by anything? Googling for it, I find some bits of old information but that doesn't seem current.

Reply:
Hmm, no wonder it had no effects on my editor. What's the new file?

Reply:
I think it depends on the editor. In vim, it even depends on the particular LSP plugin. In VScode, you apparently set it in settings.json.

Reply:
I'll just remove this then.